templates/zerver/help/include/successful-api-json-response.md

```diff
@@ -0,0 +1,9 @@
+A typical successful JSON response may look like:
+
+```
+{
```

Comment:
How universal is the actual response?  We do have many API calls that aren't dealing with a message.

Reply:
@showell: From what I could tell (I could be wrong, but I'm hoping I won't be wrong enough for these macros to be useless beyond these two cases), a success response in a case where the only information that's returned is *just* the information that signals success and nothing beyond that, the message should be universal across any such calls. Again, I could run into trouble with that as I start documenting endpoints that have never been advertised.

I guess in other words, any call that returns data that's not just an indicator of success would be different, so you are right, its not wholly universal, but I judged it to be universal enough to warrant a separate macro. :)

Reply:
I'd do one of the following:
* make the macro just be the wording part so that's consistent
* make the macro have a more specific name

I'm more inclined to the former, since I think it's good to have accurate responses when we can, especially if we can eventually have automated testing of the examples.

Reply:
I'm leaning towards the latter instead because I don't see much of a variation in accuracy when it comes to at least these two cases and I bet this isn't the only instance that the response is exactly the same! Actually, I'll document a few more endpoints and I'll go with the first one if I find that that the universality of the response is less pervasive than I initially anticipated. :)